qa/suites/fs/functional/tasks/admin.yaml

```diff
@@ -5,6 +5,7 @@ overrides:
         lockdep: true
     log-ignorelist:
       - missing required features
+      - evicting unresponsive client
```

Comment:
Why is ignoring this required? Requires an explanation in the commit message.

Reply:
Fixed

Reply:
@vshankar Fixed the commit message.